chainer/links/normalization/batch_normalization.py

```diff
@@ -84,13 +85,20 @@ def __init__(self, size, decay=0.9, eps=2e-5, dtype=numpy.float32,
         self.decay = decay
         self.eps = eps
 
-    def __call__(self, x, finetune=False):
-        """Invokes the forward propagation of BatchNormalization.
+    def __call__(self, x, **kwargs):
+        """__call__(self, x, finetune=False)
+
+        Invokes the forward propagation of BatchNormalization.
 
         In training mode, the BatchNormalization computes moving averages of
         mean and variance for evaluatino during training, and normalizes the
         input using batch statistics.
 
+        .. warning::
+
+           Do not use ``test`` argument. This argument is obsolete in v2.
```

Comment:
`test` is actually not just made obsolete but not supported anymore. How about changing this line to
```
``test`` argument is not supported anymore since v2...
```